### PR TITLE
Topic 1497 - deprecating the SWIG wrapper for MATLAB in favor of a python-based alternative.

### PR DESCRIPTION
**Pull request by Dev-iL**
Following the discussion in #1497.

### CODE REVIEW

**Comment by Dev-iL** [comment]
@jowr I have added [another commit](https://github.com/CoolProp/CoolProp/pull/1543/commits/f0447c7546499f18637b1f632982479550b21e5c) with a very minimal wrapper for `AbstractState`, and mentioned the possibility of using it in the docs.

File: wrappers/MATLAB/CoolPropsWrapperTests.m [hunk 1]
```diff
@@ -0,0 +1,115 @@
+classdef CoolPropsWrapperTests < matlab.unittest.TestCase
```
Comment (ibell) [inline wrappers/MATLAB/CoolPropsWrapperTests.m]: Can you please rename to CoolPropWrapperTests everywhere?

File: wrappers/MATLAB/README.rst [hunk 2]
```diff
@@ -1,58 +1,52 @@
...
+It is not required to follow the MATLAB's `supported python
+versions <https://www.mathworks.com/help/matlab/matlab_external/system-requirements-for-matlab-engine-for-python.html#buijfe8>`__ (this tool was tested with R2015a & R2017a + python 3.6). 
 
-    Props('FluidsList') : Returns a comma separated string of fluids in CoolProp
-    Props('version') : Returns the version of Coolprop in Use
-    Props('gitrevision') : Returns the git revision
-    Props('set_UNIT_SYSTEM_SI') : Set the unit system to SI units
-    Props('set_UNIT_SYSTEM_KSI') : Set the unit system to kilo-SI units
-    Props('get_unit_system') : Get the unit system in use, one of 'UNIT_SYSTEM_SI' or 'UNIT_SYSTEM_KSI'
+To confirm that your system is configured correctly, please execute the provided tests file using ``run(CoolPropsWrapperTests);`` and verify that it completes with no errors.
```
Comment (ibell) [inline wrappers/MATLAB/README.rst]: Change this one too to CoolPropWrapperTests

**Comment by ibell** [comment]
I cannot tell you how happy I am that you did such a nice job with the MATLAB interface to python.  It looks great, and the docs are good too.  I'm looking forward very much to dropping the current MATLAB interface, and also looking forward to dropping the builders for MATLAB that we have toiled over and fought with to get working properly.  In my day job working on REFPROP, I am very much looking forward to making that interface also use the interface you have developed.  It will drastically clean up the REFPROP interface in MATLAB.

Thanks again!!!

File: wrappers/MATLAB/CoolPropsWrapperTests.m (hunk 1 above)
Comment (Dev-iL) [reply wrappers/MATLAB/CoolPropsWrapperTests.m]: I will likely not have access to a computer for several days. Will be able to add the requested changes sometime later next week... If something changes I'll pibg you.

Comment (Dev-iL) [reply wrappers/MATLAB/CoolPropsWrapperTests.m]: ping*

**Comment by ibell** [comment]
That's fine, there's no major rush.

**Comment by Dev-iL** [comment]
@ibell Managed to get some TeamViewer time on the phone. Hopefully I hadn't missed anything....

**Pull request merged**